src/main/java/homework8/ArrayListImplementation.java

```diff
@@ -0,0 +1,130 @@
+package homework8;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ArrayListImplementation<T> {
+    private static final Object[] EMPTY_ELEMENTDATA = {};
+    private T[] typeArr;
+    private int size;
+
+    public ArrayListImplementation(int initialCapacity) {
```

Comment:
add other constructor without parameter which will constructs an empty list with an initial capacity of ten